internal/cache/metadata/stat_cache_test.go

```diff
@@ -511,7 +511,8 @@ func (t *StatCacheTest) Test_ShouldReturnHitFalseWhenOnlyObjectAlreadyHasEntry()
 
 	hit, entry := t.statCache.LookUpFolder(name, someTime)
 
-	assert.False(t.T(), hit)
+	// If "key1" object exist then corresponding folder entry will be nil, but hit will be true as key have entry in the cache for object.
```

Comment:
I didnt get this comment. aren't the keys different for objects and folders. I mean for folders they end with "/"

Reply:
got it. Ideally this scenario should never get triggered right.

Reply:
Yes.. But in test we have written like this so I wrote comment for better understanding